base_rest/components/user_component_context_provider.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 ACSONE SA/NV
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+
+from odoo.addons.component.core import AbstractComponent
+
+
+class AbstractUserComponentContextProvider(AbstractComponent):
+    _name = "abstract.user.component.context.provider"
```

Comment:
```suggestion
    _name = "abstract.user.context.provider"
```
we know already that's a component, right? :)

Reply:
Except this component provide a work context for other components, so the keyword is ComponentContext ?

Reply:
Or, we may want to align the naming with `abstract.auth.jwt.authenticated.partner.provider`. So `abstract.user.authenticated.partner.provider` ?